src/main/java/ma/internetshop/dao/jdbc/OrderDaoJdbcImpl.java

```diff
@@ -0,0 +1,207 @@
+            if (resultSet.next()) {
+                return Optional.of(getCopyOfOrder(resultSet));
+            }
+            return Optional.empty();
+        } catch (SQLException ex) {
+            LOGGER.error("Can't FIND product by ID "
+                    + id + " in mySQL", ex);
+            throw new DataProcessingException("Can't FIND product by ID "
+                    + id + " in mySQL internet_shop", ex);
+        }
+    }
+
+    @Override
+    public List<Order> getAll() {
+        String query = "SELECT * FROM orders;";
+        List<Order> orders = new ArrayList<>();
+        try (Connection connection = ConnectionUtil.getConnectionInternetShop()) {
+            Statement statement = connection.createStatement();
+            ResultSet resultSet = statement.executeQuery(query);
+            while (resultSet.next()) {
+                orders.add(getCopyOfOrder(resultSet));
+            }
+            return orders;
+        } catch (SQLException ex) {
+            LOGGER.error("Can't SELECT orders in mySQL internet_shop", ex);
+            throw new DataProcessingException("Can't SELECT orders in mySQL internet_shop", ex);
+        }
+    }
+
+    @Override
+    public Order update(Order element) {
+        String query = "UPDATE orders SET user_id = ? "
+                + "WHERE order_id = ?;";
+        try (Connection connection = ConnectionUtil.getConnectionInternetShop()) {
+            PreparedStatement statement = connection.prepareStatement(query);
+            statement.setLong(1, element.getUser().getId());
+            statement.setLong(2, element.getId());
+            statement.executeUpdate();
+            deleteOrderFromOrdersProducts(element.getId());
+            insertOrdersProducts(element);
+            LOGGER.info("Order with ID=" + element.getId() + " was UPDATED");
+            return element;
+        } catch (SQLException ex) {
+            LOGGER.error("Can't UPDATE order by ID "
+                    + element.getId()
+                    + "in mySQL internet_shop", ex);
+            throw new DataProcessingException("Can't UPDATE order by ID "
+                    + element.getId()
+                    + "in mySQL internet_shop", ex);
+        }
+    }
+
+    @Override
+    public boolean delete(Long id) {
+        String query = "DELETE FROM orders WHERE order_id = ?;";
+        return deleteById(id, query);
+    }
+
+    public boolean deleteProductFromOrder(Long id) {
+        String query = "DELETE FROM orders_products WHERE product_id = ?;";
+        return deleteById(id, query);
+    }
+
+    private boolean deleteById(Long id, String query) {
+        try (Connection connection = ConnectionUtil.getConnectionInternetShop()) {
+            deleteOrderFromOrdersProducts(id);
+            PreparedStatement statement = connection.prepareStatement(query);
+            statement.setLong(1, id);
+            int resultSet = statement.executeUpdate();
```

Comment:
```suggestion
            int ???? = statement.executeUpdate();
```
Точно ім'я змінної resultSet тут підходить?

Reply:
Да, не корректно пожалуй)

Reply:
то перейменуємо?

Reply:
ай блин

Reply:
провтыкал, извиняюсь)